hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -227,6 +228,12 @@ protected void updateIndex(Id ilId, HugeElement element, boolean removed) {
                 E.checkState(propValues.size() == 1,
                              "Expect only one property in search index");
                 value = propValues.get(0);
+
+                // for
+                if(value instanceof Collection){
+                    value = StringUtils.join(value);
+                }
```

Comment:
seems don't need to support query like textContains([text1, text2]), it's ok just to just support textContains(text) query.